contrib/devtools/symbol-check.py

```diff
@@ -276,7 +276,7 @@ def check_ELF_interpreter(binary) -> bool:
     for filename in sys.argv[1:]:
         try:
             binary = lief.parse(filename)
-            etype = binary.format.name
+            etype = binary.format
             if etype == lief.EXE_FORMATS.UNKNOWN:
```

Comment:
This also fixes matching here.

Reply:
A minor contradiction with the PR description:
> This is a minor refactor...

Reply:
Updated the description.